src/main/python/covata/delta/keystore.py

```diff
@@ -64,11 +67,10 @@ def get_private_encryption_key(self, identity_id):
         """
 
 
-class FileSystemKeyStore(DeltaKeyStore, LogMixin):
+class FileSystemKeyStore(DeltaKeyStore):
```

Comment:
I'm not sure whether that's a good idea, simply because `FileSystemKeyStore` has a defined `__init__` method, which requires its implementer to implement the same signature onwards. We can revisit this later.